ResourceStatusSystem/Utilities/InfoGetter.py

```diff
@@ -201,68 +199,87 @@ def _getComputingElementsByDomainName( targetDomain = None ):
+      if not siteCEs['OK']:
+        # return siteCEs
+        gLogger.error(siteCEs['Message'])
         continue
-      siteCEs = siteCEs[ 'Value' ]
-      ces.extend( siteCEs )
+      siteCEs = siteCEs['Value']
+      ces.extend(siteCEs)
 
   # Remove duplicated ( just in case )
-  ces = list( set ( ces ) )
-  gLogger.info( "List of CEs: %s" % str( ces ) )
+  ces = list(set(ces))
+  gLogger.info("List of CEs: %s" % str(ces))
 
-  return S_OK( ces )
+  return S_OK(ces)
 
 
-def _filterPolicies( decisionParams, policyMatchParams):
+def _filterPolicies(decisionParams, policyMatchParams):
   """
     Method that checks if the given policy doesn't meet certain conditions
   """
-  #some policies may apply or not also depending on the VO's domain
+  elementType = decisionParams.get('elementType')
+  name = decisionParams.get('name')
+
+  # some policies may apply or not also depending on the VO's domain
   # 'CEAvailabilityPolicy' can be applied only if the CE is inside LCG
-  if 'elementType' in decisionParams and 'name' in decisionParams:
-    elementType = decisionParams['elementType']
-    name = decisionParams['name']
-    if elementType and elementType.upper() == 'CE' and 'domain' in policyMatchParams:
-      #WARNING: policyMatchParams['domain'] is a list of domains
-      domains = policyMatchParams['domain']
-      result = _getComputingElementsByDomainName( targetDomain = domains )
-      if result['OK']:
-        ces = result['Value']
-        #to verify that the given CE is in the list of the LCG CEs
-        if name not in ces:
-          gLogger.info( "ComputingElement %s NOT found in domains %s" % ( name, domains )  )
-          return False
-        else:
-          gLogger.info( "ComputingElement %s found in domains %s" % ( name, domains ) )
-      else:
-        gLogger.warn( "unable to verify if ComputingElement %s is in domains %s" % ( name, domains ) )
+  if elementType and elementType.upper() == 'CE' and 'domain' in policyMatchParams:
+    # WARNING: policyMatchParams['domain'] is a list of domains
+    domains = policyMatchParams['domain']
+    result = _getComputingElementsByDomainName(targetDomain=domains)
+    if result['OK']:
+      ces = result['Value']
+      # to verify that the given CE is in the list of the LCG CEs
+      if name not in ces:
+        gLogger.info("ComputingElement %s NOT found in domains %s" % (name, domains))
         return False
+      else:
+        gLogger.info("ComputingElement %s found in domains %s" % (name, domains))
+    else:
+      gLogger.warn("unable to verify if ComputingElement %s is in domains %s" % (name, domains))
+      return False
 
   return True
 
-#EOF#EOF#EOF#EOF#EOF#EOF#EOF#EOF#EOF#EOF#EOF#EOF#EOF#EOF#EOF#EOF#EOF#EOF#EOF#EOF
+
+def postProcessingPolicyList(policiesThatApply):
+  """ Put here any "hacky" post-processing
+  """
+  if ('FreeDiskSpaceMB', 'FreeDiskSpaceMB', {}) in policiesThatApply:
```

Comment:
What is this ? Why do you need to remove all that ? Maybe you want to comment a bit more the reasons for it ? Also, should this be somewhat configurable/inheritable  ?

Reply:
This is basically a hack, as I wrote in the docstring. Nicer changes not hacky would require me some time, and they are not for a patch. What I can do is that I add a task on me.

Reply:
Sure, I understood it's a hack :-) I was just saying that you could just add a comment to say what this hack is for, so that one remembers :)